ocrd_segment/extract_regions.py

```diff
@@ -86,7 +100,7 @@ def process(self):
             ptype = page.get_type()
 
             regions = dict()
-            for name in CLASSES.keys():
+            for name in classes.keys():
                 if not name or name == 'Border' or ':' in name:
```

Comment:
I'm afraid this stopped working after I added the sub-region types to `CLASSES` for the new drawing feature of extract-pages. – This is independent of your addition.
```suggestion
                if not name or not name.endswith("Region"):
```